packages/host/src/components/Login/LoginLayout.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```

Comment:
I didn't actually change any of the styling, I just copy pasted and replaced the component with a prop in an effort to separate the ui/styling and the more logically stuffs

Reply:
it does make the jsx easier to read here, having the components reduced down like this